golem/network/transport/msg_queue.py

```diff
@@ -90,12 +107,18 @@ def waiting() -> typing.Iterator[str]:
 
 @decorators.run_with_db()
 def sweep() -> None:
-    """Sweep ancient messages"""
+    """Sweep messages"""
     with READ_LOCK:
-        oldest_allowed = datetime.datetime.now() \
+        count = 0
+        count += model.QueuedMessage.delete().where(
+            model.QueuedMessage.deadline <= default_now()
+        ).execute()
+
+        oldest_allowed = default_now() \
```

Comment:
Avoid possible inconsistency, by calling `default_now` only once.
```suggestion
        now = default_now()
        count = 0
        count += model.QueuedMessage.delete().where(
            model.QueuedMessage.deadline <= now
        ).execute()

        oldest_allowed = now \
```

Reply:
Good catch, will do.